Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM node:16-alpine as base
```

Comment:
```suggestion
FROM node:18-alpine as base
```

Reply:
I would do this as part of this PR, but I agree we should update this